docs/dbus/filesystem.xml

```diff
@@ -98,4 +98,40 @@
       <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const" />
     </property>
   </interface>
+  <interface name="org.storage.stratis3.filesystem.r2">
+    <!-- Set the filesystem name. -->
+    <method name="SetName">
+      <!-- The new name of the filesystem. -->
+      <arg name="name" type="s" direction="in" />
+      <!--
+        b: True if the name was changed, otherwise false.
+        s: UUID of the filesystem or a default UUID if no change was made.
+      -->
+      <arg name="result" type="(bs)" direction="out" />
+      <arg name="return_code" type="q" direction="out" />
+      <arg name="return_string" type="s" direction="out" />
+    </method>
+    <!-- Time the filesystem was created. -->
+    <property name="Created" type="s" access="read">
+      <annotation name="org.freedesktop.DBus.Property.EmitsChangedSignal" value="const" />
+    </property>
+    <!-- Device node of the filesystem's thin device. -->
```

Comment:
This may be a little bit unclear to users. Do you think something like "Path to filesystem devnode" might be a little clearer?

Reply:
I'ld rather leave this because we called it the same in r0 and r1.

Reply:
I think it's unnecessarily confusing to mention the thin device. It is irrelevant to the API that it is on a thin device and I think less is more here.